src/kem/bike/additional/secure_decode_avx2.c

```diff
@@ -31,7 +31,7 @@ rotate256_big(OUT syndrome_t *out, IN const syndrome_t *in, IN size_t ymm_num) {
 	bike_static_assert(sizeof(*out) > (YMM_SIZE * (R_YMM + (2 * R_YMM_HALF_LOG2))),
 	                   rotr_big_err);
 
-	const __m256i all_one_mask = _mm256_set1_epi8(0xff);
+	const __m256i all_one_mask = _mm256_set1_epi8((char) 0xff);
```

Comment:
Why did you add this `(char)` cast everywhere? Which warning/compiler/os triggers this change? 
In general, if BIKE's code is changed without updating me or Shay, we may override the change next time we merge BIKE's repository. It is best to update the code there and then merge.